src/main/java/com/iwaconsolti/league/controller/PlayerController.java

```diff
@@ -0,0 +1,34 @@
+package com.iwaconsolti.league.controller;
+
+import com.iwaconsolti.league.model.PlayerModel;
+import com.iwaconsolti.league.service.PlayerService;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import java.util.List;
+
+@RestController
+@RequestMapping("/Player")
+@Slf4j
+public class PlayerController {
+
+    private final PlayerService service;
+    @Autowired
+    public PlayerController(PlayerService service) {
+        this.service = service;
+    }
+    @GetMapping("/{teamid}")
```

Comment:
Using this mapping, /Player/1 i will think that I'm returning the information of player 1 not the players of Team 1